Sources/Purchasing/StoreKit2/ProductsFetcherSK2.swift

```diff
@@ -94,7 +94,7 @@ private extension ProductsFetcherSK2 {
         // Note: not using `Storefront.updates` because it's not currently possible
         // to avoid race conditions. Updates never arrive on time for tests to detect
         // changes and clear the cache.
-        get async { await Storefront.current?.id }
+        get async { await StoreKit.Storefront.current?.id }
```

Comment:
Thoughts on adding an extension to `Storefront` like this?
```swift
public extension Storefront {

    @available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, macCatalyst 13.1, *)
    private static var currentStorefrontType: StorefrontType? {
        get async {
            if #available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *) {
                let sk2Storefront = await StoreKit.Storefront.current
                return sk2Storefront.map(SK2Storefront.init)
            } else {
                return SKPaymentQueue.default().storefront.map(SK1Storefront.init)
            }
        }
    }

    /// The current App Store storefront for the device.
    @available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, macCatalyst 13.1, *)
    static var current: Storefront? {
        get async {
            return await self.currentStorefrontType.map(Storefront.from(storefront: ))
        }
    }

}
```

Then changing this to `Storefront.current` and users can use it too while supporting SK1.

Reply:
Makes sense to me 😄

Reply:
I'll add it in another PR.

Reply:
Done in #1502